client/src/components/ShardComponents/Login/index.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react"
+import "./style.css"
+class Login extends Component {
+  state = {
+    email: "",
+    password: "",
+    status: "false"
+  }
+
+  handelChange = event => {
+    const value = event.target.value
+    this.setState({ [event.target.name]: value })
+  }
+
+  render() {
+    return (
+      <form onSubmit={this.submit}>
+        <label htmlFor="email"></label>
+        <input
+          placeholder="Email"
+          type="email"
+          id="email"
+          name="email"
+          value={this.state.email}
+          onChange={this.handelChange}
+        />
+        <br />
```

Comment:
don't use ```<br />``` to add spaces between elements. This is designed to add line breaks for texts.

use margins instead.